product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -280,3 +282,41 @@ A SQL Server URL contains the following information:
 -   `SRC_DB_USER` should specify the name of a privileged SQL Server user.
 
 -   `SRC_DB_PASSWORD` must contain the password of the specified user.
+
+## Defining Target Information for EDB Cloud
```

Comment:
Can we move this up a bit in the doc? Ideally more folks come here looking for EDB Cloud than, say, Sybase :)

Reply:
I think EDB Cloud should be before MySQL and SQL Server as well.  Maybe even before Oracle.

Reply:
I moved it to before Oracle so the sections talking about the target of the migration are grouped and the sections talking about the source of the migration are grouped.

Reply:
I also changed the name of the section and updated the intro of the section a bit. Is this correct? Should we remove SQL Server and MySQL from the list?

**Defining an EDB Cloud URL**
Migration Toolkit facilitates migration from the following platforms to EDB Cloud:

- Oracle
- MySQL
- SQL Server
- Advanced Server

Reply:
Should we move Advanced Server up in the list before MySQL and Microsoft SQL Server?  Also, instead of saying "Advanced Server" should we be saying "EDB Postgres Advanced Server".  Regarding the ordering of the list, we should either order the source alphabetically  or in order of our level of support.  Interestingly, if "SQL Server" is changed to "Microsoft SQL Server", I think the level of support ordering and the alphabetic ordering would match.